### PR TITLE
Update configure-endpoints-vdi.md

### LINKED ISSUE #13211: Benefit of using PSExec?
Page feedback 26becb70-47ff-38fe-592f-99d84f6aba12
https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/configure-endpoints-vdi?view=o365-worldwide#updating-virtual-desktop-infrastructure-vdi-images-persistent-or-non-persistent

What is the benefit of using PSExec when the instructions are relating to local computer?
image

It adds unnecessary confusion.

Regards

### PR DESCRIPTION
**Pull request by msbemba**
Removed psexec as instructions are related to a local computer

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/13211

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-endpoints-vdi.md [hunk 1]
```diff
@@ -160,10 +160,9 @@ If you have onboarded the primary image of your VDI environment (SENSE service i
    sc query sense
    ```
 
-3. Run the following commands using PsExec.exe (which can be downloaded from [https://download.sysinternals.com/files/PSTools.zip](https://download.sysinternals.com/files/PSTools.zip)):
+3. Run the following commands in a CMD window::
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-vdi.md:163]:
```suggestion
3. Run the following commands in a CMD window:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Pull request merged**